src/options/modules/ColorSchemeModeHelper.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * // does not work in Firefox currently due to https://bugzilla.mozilla.org/show_bug.cgi?id=1595037
```

Comment:
no comment in comments :stuck_out_tongue_winking_eye: 

```suggestion
 * does not work in Firefox currently due to https://bugzilla.mozilla.org/show_bug.cgi?id=1595037
```